controllers/controller-babel/userController.js

```diff
@@ -80,38 +134,54 @@ const getSpacificUser = async (req, res) => {
 exports.getSpacificUser = getSpacificUser;
 
 const createNewUser = async (req, res) => {
+  const {
+    Username,
+    password,
+    Email,
+    Fullname
+  } = req.body;
+  const emailIsVerified = false;
+  const {
+    error
+  } = registerValidation({
+    Email,
+    password,
+    Username,
+    Fullname
+  });
+  if (error) return res.status(400).json({
+    "error": error.details[0].message
+  });
+
   try {
-    const {
-      Username,
-      password,
-      Email,
-      userId,
-      emailIsVerified,
-      Fullname
-    } = req.body;
     const salt = await _bcrypt.default.genSalt(10);
-    const hashedPassword = await _bcrypt.default.hash(password, salt); // validations will happen here
-    // if(newUsers.trim() === '' || newUsers.trim() === null){
-    //     res.status(400).json({'message' : "Please make sure you have provided user information"});
-    //     return;
-    // }
+    const hashedPassword = await _bcrypt.default.hash(password, salt);
+    /* ===== Start:: making sure email is unique ====== */
+
+    const emailExist = await _Users.default.findOne({
+      Email: Email
+    });
+    if (emailExist) return res.status(400).json({
+      "error": "Email already exist "
+    });
+    /* ====== End:: making sure email is unique ======= */
 
     const newUser = new _Users.default({
       Username,
       Password: hashedPassword,
       Email,
-      userId,
       emailIsVerified,
-      Fullname
+      Fullname,
+      userType: "normal"
     });
     const savedUser = await newUser.save();
     res.status(200).json({
-      savedUser
+      "userId": savedUser._id
     });
   } catch (error) {
     console.log(error);
     res.status(500).json({
-      "message": "Server error"
+      "error": "Server error"
```

Comment:
be consistent in your return messages, 
if it is `error`: `any message`, let's stick to that on all endpoints, it will be easier in the front-end to know what to expect